tests/unit/admin/admin-gutenberg-compatibility-notification-test.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * WPSEO plugin test file.
+ *
+ * @package WPSEO\Tests\Admin
+ */
+
+namespace Yoast\WP\SEO\Tests\Unit\Admin;
+
+use Brain\Monkey;
+use Yoast\WP\SEO\Tests\Unit\Doubles\Admin\WPSEO_Admin_Gutenberg_Compatibility_Notification_Double;
+use Yoast\WP\SEO\Tests\Unit\TestCase;
+
+/**
+ * Unit test class
+ *
+ * @group MyYoast
+ *
+ * @coversDefaultClass WPSEO_Admin_Gutenberg_Compatibility_Notification
+ */
+class WPSEO_Admin_Gutenberg_Compatibility_Notification_Test extends TestCase {
+
+	/**
+	 * Holds the WPSEO_Gutenberg_Compatibility mock instance.
+	 *
+	 * @var WPSEO_Gutenberg_Compatibility
+	 */
+	private $gutenberg_compatibility_mock;
+
+	/**
+	 * Holds the Yoast_Notification_Center mock instance.
+	 *
+	 * @var Yoast_Notification_Center
+	 */
+	private $notification_center_mock;
+
+	/**
+	 * Set up the mocked dependencies.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->gutenberg_compatibility_mock = $this
```

Comment:
Ideally creating these mocks would use `Mockery::mock` for consistency with the majority of our other unit tests.

Reply:
The mocks are now created with `Mockery::mock` in commit 9c7411c